packages/eip1193-permission-middleware/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@metamask/chain-agnostic-permission` to `^0.7.0` ([#5818](https://github.com/MetaMask/core/pull/5818))
```

Comment:
Nit: what do you think about merging three lines to bump chain agnostic permission into one entry with with comma space separated prs ?

Reply:
cleaned up here: https://github.com/MetaMask/core/pull/5818/commits/e967dd7e2abb6cd5eeabc5c5745b4a8f2f0f5d9a